File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By


with webdriver.Firefox() as driver:

    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    print("title is : ", driver.title)

    driver.find_element(By.ID, "prompt").click()
    prompt_alert =driver.switch_to.alert

    print("the alert text is : " ,prompt_alert.text)

    prompt_alert.send_keys("Awsome!!")
    print("user entred the text in prompt alert")
    prompt_alert.accept()

    driver.quit()

File: Selenium/Activities/Activity20.java
package activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity20 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/javascript-alerts");
		System.out.println("The Title of the page is  : " + driver.getTitle());

		driver.findElement(By.id("prompt")).click();

		Alert alt = driver.switchTo().alert();

		String altText = alt.getText();
		System.out.println(altText);
		
		alt.sendKeys("Awsome!!");
		alt.accept();
		
		driver.quit();

	}

}
